R/metadata.R

```diff
@@ -118,6 +123,66 @@ get_barchart_settings <- function() {
   )
 }
 
+get_calibration_plot_settings <- function(filter_types) {
+  calibrate_only_settings <- list(
+    list(
+      filterId = scalar("calibrate_indicator"),
+      label = scalar(get_label_for_id("calibrate_indicator")),
+      stateFilterId = scalar("indicator")
+    )
+  )
+  filterIds <- c("spectrum_region", "type", "period", "sex", "age")
+  list(
+    defaultEffect = list(
+      setFilters = c(calibrate_only_settings,
+                     lapply(filterIds, get_filter_from_id)),
+      setFilterValues = list(
+        indicator = c("prevalence"),
+        period = get_filter_option_ids(filter_types, "period")[2],
+        sex = get_filter_option_ids(filter_types, "sex")[1],
+        age = c("Y015_049"),
+        calibrate_type = get_filter_option_ids(filter_types, "type"),
+        spectrum_region = get_filter_option_ids(filter_types, "spectrum_region")
+      ),
+      setHidden = c(
+        "type", "spectrum_region"
+      )
+    ),
+    ## x-axis and disaggregate plot settings are not visible as users cannot
+    ## change these in the calibrate plot
+    plotSettings = list(
+      list(
+        id = scalar("x_axis"),
+        label = scalar(t_("OUTPUT_BARCHART_X_AXIS")),
+        options = list(get_x_axis_or_disagg_by_option("spectrum_region")),
+        visible = scalar(FALSE)
```

Comment:
minor suggestion, perhaps a very low priority ticket but would be good to be consistent with this, you have a `setHidden` effect so would be nice if this boolean was `hidden` instead of `visible` but super lower priority so fine to ignore

Reply:
Thanks, yep let's be consistent here